packages/react-components/package.json

```diff
@@ -86,7 +93,7 @@
     "@vaadin/virtual-list": "24.4.0-alpha11"
   },
   "author": "Vaadin Ltd.",
-  "license": "Apache-2.0",
+  "license": "(Apache-2.0 OR SEE LICENSE IN https://vaadin.com/commercial-license-and-service-terms)",
```

Comment:
Curious to see if that works, but it looks correct according to NPM docs.

Reply:
This is also what `@vaadin/vaadin` uses https://www.npmjs.com/package/@vaadin/vaadin